app/services/post_alerter.rb

```diff
@@ -170,7 +171,7 @@ def after_save_post(post, new_record = false)
 
       if topic.present?
         watchers = category_watchers(topic) + tag_watchers(topic) + group_watchers(topic)
-        notify_first_post_watchers(post, watchers)
+        notify_first_post_watchers(post, watchers, notified)
```

Comment:
Should we also update the notified array do it's accurate later on?

```suggestion
        notified += notify_first_post_watchers(post, watchers, notified)
```

Reply:
No one else reads the notified array at a later stage. I did not want to perform an operation whose result will never be used.

Reply:
How about those who listens to the `post_alerter_after_save_post` event?

Reply:
That is fair. :+1: